### ISSUE TITLE
Consensus-layer Call 140

### ISSUE THREAD

**Issue by ralexstokes** [ethereum/pm#1129]
# Consensus-layer Call 140

[prev: call 139](https://github.com/ethereum/pm/issues/1116)

Meeting Date/Time: [Thursday 2024/8/22 at 14:00 UTC](https://savvytime.com/converter/utc/aug-22-2024/2pm)
Meeting Duration: 1.5 hours
[stream](https://youtube.com/live/hSNJ6vURfmE?feature=share)

0. https://github.com/ethereum/consensus-specs/releases/tag/v1.5.0-alpha.5
1. Electra
    * devnet-2 and devnet-3 status
    * https://github.com/ethereum/consensus-specs/pull/3882
    * Migrate execution requests out of `ExecutionPayload` in CL data type
    * Unify requests representation in the Engine API
2. PeerDAS
    * implementation updates?
    * Blob uncoupling b/t EL and CL: EIP-7742 -- Pectra inclusion?
    * Blob configurability in clients
    * other open spec questions?
3. Research, spec, etc.
    * F-star name
    * Probelab update on gossip analysis
4. Open discussion/Closing remarks

**Comment by ralexstokes** [comment]
I'd like to revisit the name of the F-star for the hard fork after Pectra.

Results integrated from various sources here: https://ethereum-magicians.org/t/f-star-name-for-consensus-layer-upgrade-after-electra/20285

Would suggest we go with `Fulu` and use the portmanteau `Fosaka`

**Comment by lucassaldanha** [comment]
I would like to get some feedback on this PR based on @potuz's proposal of moving the execution layer request out of the execution payload.

TLDR: we move `deposit_requests`, `withdrawal_requests`, and `consolidation_requests` into a new container `ExecutionLayerRequests`, and make them part of `BeaconBlockBody`, removing them from `ExecutionPayload`. This "restores" the assumption that the full Execution Payload isn't necessary for the CL performing state transition.

This issue has more context and some discussion: https://github.com/ethereum/consensus-specs/issues/3856

And this is the PR with the proposed changes: https://github.com/ethereum/consensus-specs/pull/3875

**Comment by mkalinin** [comment]
Want to give a quick overview of the following change:
* https://github.com/ethereum/consensus-specs/pull/3882

**Comment by cortze** [comment]
I'd like to present a summary of the results that [ProbeLab](https://probelab.io/) has done on the GossipSub domain. I will present our most relevant findings and the main takeaways from the study.   
Of course, all feedback and suggestions for further studies and improvement will be highly appreciated.

**Comment by ralexstokes** [comment]
blob flexibility in clients -- is the limit a compile-time constant? how easy is it to change?

TODO: latest on EL/CL blob uncoupling

**Comment by lightclient** [comment]
Hello can we have a final check on https://github.com/ethereum/execution-apis/pull/565

**Comment by etan-status** [comment]
Not in call myself, but would like to bring up:

- https://github.com/ethereum/consensus-specs/pull/3885 -- 
  - CLs may not do newPayload on every single block (to avoid engine API overhead), or may even support optimistic syncing without an EL (maintenance etc)
  - These CLs still need to do minimal checks on every block that doesn't go through newPayload, to ensure that the execution data in the beacon block is linked with the `body.execution_payload.block_hash` (so that it is guaranteed that everything building on this `block_hash` is also INVALID if there are asynchronous errors found later on by the EL). The minimal checks are:
    - No tx can be `[]` (the one added in the issue)
    - EL block hash must match the execution data
    - `blob_kzg_commitments` must match the `blob_versioned_hashes` from EIP-4844 blob transactions
  - The empty tx check is already present in most ELs, so no immediate rush. But still advising CLs to check their impls, even though this only affects non-validating clients (validating clients use `newPayload` and therefore have the EL perform the checks already).
- Generally, the pitfalls with `newPayload` above are prompted by reluctance to move forward with issues such as SSZification. Aligning data formats across CL, EL, and engine API, while not being visible changes externally, are still important to keep complexity from growing.
- EIP-7688 CL StableContainer kurtosis config updated for latest Lodestar, would be great to also add grandine, lighthouse, prysm eventually (config @ https://stabilitynow.box) 
- EIP-7732 enshrined proposer / builder separation: Personally in support of it (also benefits from SSZ 'optional's)

**Comment by akashkshirsagar31** [comment]
Podcast (audio only) - https://open.spotify.com/episode/7gPCCsgSq6LCDzt0qHbyFL?si=n40e3HS3T_2KrThfn_bC2w

**Comment by ralexstokes** [comment]
closing in lieu of https://github.com/ethereum/pm/issues/1140
